packages/app/src/debug/DebugArtifactLink.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <Tooltip
+    placement="bottom"
+    :data-cy="`artifact-for-${icon}`"
+  >
+    <ExternalLink
+      class="flex h-full w-full items-center justify-center"
+      :data-cy="`${icon}-button`"
+      :href="props.url || '#'"
+      :use-default-hocus="true"
```

Comment:
```suggestion
      :use-default-hocus="true"
      :aria-label="popperText"
```

This will provide audio description for screenreader users and may reduce the need for data-cy attributes, we can do a `cy.findByRole('link', { name: 'View Log' })`, which is a bit more intuitive in the tests too.